03_backend_developement/01_HTTP/01_get_the_weather/.__tests__/03_berlin_weather_in_german.test.js

```diff
@@ -0,0 +1,28 @@
+const fs = require("fs");
+const path = require("path");
+const curl = require("./curl");
+const exec = require("./exec");
+
+const file = () => path.resolve(__dirname, "../03_berlin_weather_in_german.sh");
+
+test("the script should return the weather of Berlin", done => {
+  expect.assertions(1);
+
+  Promise.all([exec(`bash ${file()}`), curl("de.wttr.in", "/Berlin")]).then(
```

Comment:
do we need "our" curl if we don't use the result?

Reply:
No, I just made the test after and copy/pasted ^^'